doc/commands.md

```diff
@@ -0,0 +1,33 @@
+# CLI commands documentation
+
+This document will cover and document three commands, attach, detach, and list. All three commands have a a log variable as well as two functions, get_parser, and take_action. 
+
+For each function the get_parser method is the same:
+* The function can add a node, network, or port, depending on the class. 
+* The parser is then returned. 
+
+Links to the commands are here:
+* [Attach](#attach)
+* [Detach](#detach)
+* [List](#list)
+
+
+## <a name="attach"></a>Attach
+
+The attach command will attack a network to a node. 
+
+*`esi_node_network_attach node --port --network`, a node is required while a port and a network are optional.
```

Comment:
The command a user would actually use is 'openstack esi node network attach'. I might display it as follows:

* `openstack esi node network attach <node> --port <port> --network <network>`
      * node: name or UUID of the node
      * port: name or UUID of the port
 
etc

You can get some information about what the parameters are in the code; I think there are helpstrings in there. Also note that the user has to specify exactly one of port or network.